spec/EventListener/OrderCreationListenerSpec.php

```diff
@@ -43,10 +43,10 @@ function it_completes_order_before_creation(
         $event->getSubject()->willReturn($order);
 
         $stateMachineFactory->get($order, 'sylius_order_checkout')->willReturn($stateMachine);
-        $stateMachine->apply(OrderCheckoutTransitions::TRANSITION_ADDRESS)->shouldBeCalled();
-        $stateMachine->apply(OrderCheckoutTransitions::TRANSITION_SELECT_SHIPPING)->shouldBeCalled();
-        $stateMachine->apply(OrderCheckoutTransitions::TRANSITION_SELECT_PAYMENT)->shouldBeCalled();
-        $stateMachine->apply(OrderCheckoutTransitions::TRANSITION_COMPLETE)->shouldBeCalled();
+        $stateMachine->can(OrderCheckoutTransitions::TRANSITION_ADDRESS)->shouldBeCalled();
+        $stateMachine->can(OrderCheckoutTransitions::TRANSITION_SELECT_SHIPPING)->shouldBeCalled();
+        $stateMachine->can(OrderCheckoutTransitions::TRANSITION_SELECT_PAYMENT)->shouldBeCalled();
+        $stateMachine->can(OrderCheckoutTransitions::TRANSITION_COMPLETE)->shouldBeCalled();
```

Comment:
I believe `apply(...)->shouldBeCalled()` should still be there. More, all of these should be `$stateMachine->can(...)->willReturn(true);` We should also create at least one spec when some of the transition cannot be called (and check that `apply(...)->shouldNotBeCalled()`